src/main/java/com/faforever/client/patch/GameBinariesUpdateTaskImpl.java

```diff
@@ -133,7 +133,11 @@ void copyGameFilesToFafBinDirectory() throws IOException {
 
             logger.debug("Copying file '{}' to '{}'", source, destination);
             noCatch(() -> createDirectories(destination.getParent()));
-            noCatch(() -> copy(source, destination, REPLACE_EXISTING));
+            noCatch(() -> {
+              if (!Files.exists(destination)) {
+                copy(source, destination, REPLACE_EXISTING);
+              }
```

Comment:
this solves the issues with the game not terminating correctly... just so u understand.

Reply:
got it